### DISCUSSION TITLE
Synology NAS Docker error

### DISCUSSION THREAD

**Discussion by zeister** [mp-se/fermentrack-docker#25] [General]
When i try to run your image on my  Synology NAS, I  am getting an error that  says "Volume  for  data and / or database is NOT mounted. Aborting startup ..." Any help would be greatly appreciated!

**Comment by mp-se** [comment]
You have not specified the required volumes for the data and db, see the installation instructions. This will depend on what your host is so if you can share more about your setup I can point you in the right direction 

https://fermentrack-docker.readthedocs.io/en/latest/installation.html
